### ISSUE TITLE
DateTimePicker keyboard navigation

### ISSUE THREAD

**Issue by mlp73** [jquense/react-widgets#981]
Hi!
I cannot find a way to correctly access the DateTimePicker when navigating with keyboard. I read the following in the documentation and make sure the component has an ID.

"Each widget also has appropriate ARIA roles and attributes for the benefit of screen readers and visually impaired users. **Keyboard-only navigation of widgets is also supported**, for those who prefer to not, or cannot use a mouse. To help ensure maximum accessibility, **every widget should have an id attribute**"

Despite that I can see that the button (.rw-btn .rw-btn-select) opening the Datepicker is systematically receiving a tabindex -1 which makes it impossible to navigate to.
Is there any prop to manage that, or something obvious I did not find?

Regards

**Comment by jquense** [comment]
Click the ![image](https://user-images.githubusercontent.com/339286/70716347-382a1c00-1cba-11ea-8c00-b63f15e83bad.png) button on the doc page http://jquense.github.io/react-widgets/api/DateTimePicker/ for a list of the shortcuts
